Source/Initialization/WarpXInitData.cpp

```diff
@@ -1272,12 +1263,12 @@ void WarpX::InitializeEBGridData (int lev)
                     m_eb_update_B[lev],
                     m_fields.get_alldirs(FieldType::Bfield_fp, lev),
                     eb_fact );
-
             }
 
         }
 
         ComputeDistanceToEB();
+        MarkReducedShapeCells( m_eb_reduce_particle_shape[lev], eb_fact, WarpX::nox );
```

Comment:
I think you don't need to pass `m_eb_reduce_particle_shape` (or even all) arguments here and could call it like `ComputeDistanceToEB()`. (Because both are member functions of the same class.)

Reply:
Sounds good. 
Part of the motivation here is to maybe do a follow-up PR where some of the `Mark...Cells` functions become free functions.